docker/k8s/docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM sammyne/hyperledger-tcf-vm:alpha 
+
+LABEL maintainer=lixiangmin01@baidu.com
+
+RUN apt update -y && apt install -y vim
+# the sed cmd belows is to decrease the 1TB LMDB cache size for local development
+RUN cd /project && rm -rf TrustedComputeFramework &&\
+    git clone --branch k8s --depth 1 https://github.com/sammyne/trusted-compute-framework.git &&\
```

Comment:
You should be cloning the main repository, `https://github.com/hyperledger-labs/trusted-compute-framework.git`

Reply:
done as 7f48d2d0a38818265784e988749c61d1b993add2